src/js/pieChart.js

```diff
@@ -68,21 +72,41 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
         },
         selectors: {
             pie: ".floec-ca-pieChart-pie",
-            slice: ".floec-ca-pieChart-legend"
+            legend: ".floec-ca-pieChart-legend"
+        },
+        listeners: {
+            "onCreate.renderTemplate": "floe.chartAuthoring.pieChart.renderTemplate"
```

Comment:
Both good ideas - other components making use of the templateInjection grade in the future may also require an event to indicate the template content has been injected.